editor/styling/stylesheets/modules/_properties.scss

```diff
@@ -20,12 +24,21 @@
             -fx-min-width: 24px;
             -fx-background-radius: 0;
             -fx-border-radius: 0;
-            -fx-background-color: transparent;
             -fx-border-color: transparent;
             visibility: hidden;
 
+            &.drag-handle {
+                -fx-max-width: 16px;
+                -fx-min-width: 16px;
+            }
+
+            &.color-dropper {
+                -fx-background-color: transparent;
+            }
```

Comment:
I removed this rule from the handle. Now that its width is a lot smaller, having a background helps us grab the handle, because the constrains are visible.